### ISSUE TITLE
Google Safe Browsing falsely determines opendrr.github.io as "Deceptive site" (detected phishing)

### ISSUE THREAD

**Issue by anthonyfok** [OpenDRR/opendrr-platform#119]
Around midnight of March 13, 2021, when trying to access https://opendrr.github.io/ from Mozilla Firefox or Google Chrome, the following scary-looking all-red "Security error" appears:

![Screenshot from 2021-03-13 02-44-18](https://user-images.githubusercontent.com/1274764/111026260-9c003080-83a6-11eb-9fd7-6eed22f88c75.png)

Clicking on the "detected phishing" link enters https://transparencyreport.google.com/safe-browsing/search?url=https:%2F%2Fopendrr.github.io%2F&hl=en-GB as shown below:

![Screenshot from 2021-03-13 02-44-46](https://user-images.githubusercontent.com/1274764/111026271-a6bac580-83a6-11eb-80e2-a22c25abda18.png)

I am gathering some more data before I fill in the "Report Incorrect Phishing Warning" page at https://safebrowsing.google.com/safebrowsing/report_error/?url=https%3A%2F%2Fopendrr.github.io%2F&hl=en-GB

**Comment by anthonyfok** [comment]
* Sucuri: "No Malware Found", "Site is not Blacklisted"
    * https://sitecheck.sucuri.net/results/opendrr.github.io
* Quttera: "Potentially Suspicious files: 2"
    * ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js (Detected potentially suspicious initialization of function pointer to JavaScript method)
    * wet-boew.github.io/themes-dist/GCWeb/GCWeb/js/theme.min.js (Potentially Suspicious, "PS.JS.Obfuscantion.gen")
    * See https://quttera.com/detailed_report/opendrr.github.io

VirusTotal: 3 engines (out of 85) detected issues with this URL.

* https://www.virustotal.com/gui/url/419e325c8aac45e2422075e15c3561c00ea43e4683ae6c955afefec5a1ef4f81/detection
    * Avira (no cloud) detects "Phishing"
    * Emsisoft detects "Phishing"
    * Netcraft detects "Malicious"
    * Quttera detects "Suspicious"

* Netcraft:
    * https://sitereport.netcraft.com/?url=https%3A%2F%2Fopendrr.github.io%2F

Sounds like false positive to me.

....

Anyhow, I submitted the following report via https://safebrowsing.google.com/safebrowsing/report_error/?url=https%3A%2F%2Fopendrr.github.io%2F&hl=en-GB:

> Hello,
> 
> OpenDRR (Open Disaster Risk Reduction) Platform is a project under Natural Resources Canada, Government of Canada.  It is middleware between hazard or risk modeling environments like OpenQuake and end users who need to understand and evaluate risk to make economic and policy decisions.  Please see https://github.com/OpenDRR/opendrr-platform for more information.
> 
> The https://opendrr.github.io/ is being served on GitHub Pages from three repos, namely:
> 
> * https://github.com/OpenDRR/opendrr.github.io
> * https://github.com/OpenDRR/data
> * https://github.com/OpenDRR/documentation
> 
> The web pages, though incomplete, are up for demo and internal development.  Besides rendered HTML files (from Markdown and AsciiDoc), OGC GeoPackage format data files (*.gpkg) for Canadian provinces and territories are offered for download.  There are also some design documentation in PDF format, though not in GitHub Pages.  Dynamic maps will be made available in the near future.
> 
> The above resources are all that is hosted as Jekyll-based statically generated website on https://opendrr.github.io/, so there is no phishing going on at https://opendrr.github.io/ to the best of our knowledge.  Thank you.
> 
> Yours sincerely,
> 
> Anthony Fok <<anthony.fok@canada.ca>>
> on behalf of OpenDRR, Natural Resources Canada

**Comment by anthonyfok** [comment]
Wrote the following from incognito window:

> This is an official Government of Canada project (under Geological Survey of Canada - Natural Resources Canada), and while not using a .ca domain yet, is definitely NOT a phishing site.  Thank you!

**Comment by anthonyfok** [comment]
Request Review via "Google Search Console" Tools (formerly Google Webmasters Tools last time I used it)

**Comment by anthonyfok** [comment]
Some relevant links that might be helpful?

* [Social Engineering (Phishing and Deceptive Sites)](https://developers.google.com/search/docs/advanced/security/social-engineering)
* [Google Search Console Security Issues Explained](https://www.searchenginejournal.com/google-search-console-security-issues-explained/367918/#close)
* [How To Remove ‘Deceptive Site Ahead’ Warning? - MalCare](https://www.malcare.com/blog/deceptive-site-ahead/)

Could the issue be resolved within 24 hours?  Or within 1 - 3 days?  I guess the real problem is that we do not really know what exactly triggered Google's false alarm, especially it says "Sample URLs: N/A" ...  It couldn't be just because we are using GCWeb theme and Canada.ca boilerplate on a non canada.ca domain, right?

![Screenshot from 2021-03-13 09-22-54](https://user-images.githubusercontent.com/1274764/111036716-df29c600-83dd-11eb-9ec0-c543ad197ad0.png)

**Comment by anthonyfok** [comment]
Thanks be to God!

And "Thanks be to Google" especially for the quick response of their staff.
I thought we had to wait for days for this to be resolved, but thankfully, only a few hours.

I am happy to report that, as of about noon today (March 13), our https://opendrr.github.io/ is no longer listed as a "Deceptive site", and is back to normal on all browsers.  We are off Google's blacklist!  Hurray!

Here is a screenshot of Google giving our website a clean bill of health:

![Screenshot from 2021-03-13 15-02-51](https://user-images.githubusercontent.com/1274764/111051048-9427a700-840d-11eb-98c1-09af5ea09441.png)

Closing this issue.  (Phew!)
